annsa/tests/test_readspectra.py

```diff
@@ -0,0 +1,14 @@
+import os
+from annsa.annsa import read_spectrum
+
+
+def test_read_spectrum1():
```

Comment:
```suggestion
def test_read_spectrum_rocky_flats():
```

Reply:
I recommend the more meaningful name, above.
More importantly, though, this test simply tests whether this function runs without throwing an exception.
A real test would check that the resulting object, `_`, has the features you expect, given you know the file it's read.

(Related... why are variables that are plain underscores used everywhere in this codebase? It's unusual to me.)

Reply:
The plain underscore is the convention I use. If there is a more widely accepted convention I can switch to that.

Reply:
Latest commit tests equivalency.